bin/deps.ml

```diff
@@ -0,0 +1,44 @@
+(*
+ * Copyright (c) 2020 Ulysse Gérard <ulysse@tarides.com>
+ *
+ * Permission to use, copy, modify, and distribute this software for any
+ * purpose with or without fee is hereby granted, provided that the above
+ * copyright notice and this permission notice appear in all copies.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ * ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+ *)
+
+let run files =
+  List.iter (fun file ->
+    let syntax = match Mdx.Syntax.infer ~file with
+    | Some s -> s
+    | None -> Mdx.Normal
```

Comment:
Shouldn't this command take `--syntax` ? The caller may know better about this than our `Syntax.infer`.
Also, I think we should exit with an error in this case, instead of guessing.